File: app/src/main/java/com/returnnotfound/stain/base/activity/IBaseActivityView.java
package com.returnnotfound.stain.base.activity;

import android.content.Context;

import com.returnnotfound.stain.base.IBaseView;
import com.returnnotfound.stain.base.fragment.BaseFragment;

public interface IBaseActivityView extends IBaseView {
  BaseFragment createRootFragment();

  int getContainerId();

  Context getActivityContext();
}

File: app/src/main/java/com/returnnotfound/stain/base/activity/IBaseActivityView.kt
package com.returnnotfound.stain.base.activity

import com.returnnotfound.stain.base.IBaseView
import com.returnnotfound.stain.base.fragment.BaseFragment

interface IBaseActivityView : IBaseView {
  fun getRootFragment(): BaseFragment

  fun getContainerId(): Int
}